pmd-core/src/main/java/net/sourceforge/pmd/cpd/CPD.java

```diff
@@ -59,6 +63,10 @@ public Iterator<Match> getMatches() {
         return matchAlgorithm.matches();
     }
 
+    public Map<String, Integer> getNumberOfTokensPerFile() {
```

Comment:
This is now accessible from the report

```suggestion
    private Map<String, Integer> getNumberOfTokensPerFile() {
```

Reply:
I think the method can even be removed then; we can directly use the `numberOfTokensPerFile` field.